modules/apps/object/object-rest-api/src/main/java/com/liferay/object/rest/manager/v1_0/DefaultObjectEntryManager.java

```diff
@@ -76,6 +77,12 @@ public ObjectEntry fetchRelatedManyToOneObjectEntry(
 			String objectRelationshipName)
 		throws Exception;
 
+	public BaseModel<?> fetchRelatedManyToOneSystemObjectEntry(
```

Comment:
We are returning here a BaseModel, but in the rest of methods we return `Object`. Does it make sense?

Reply:
This method is not being used from anywhere. Do we need to expose it?